erts/test/erlc_SUITE.erl

```diff
@@ -435,6 +487,613 @@ make_dep_options(Config) ->
+        ?OK([])),
+
+    %% File has quoted atoms which are keywords in experimental_ftr_2.
+    %% We should see no warnings.
+    Run("foo.erl", options([longopt(enable, experimental_ftr_2),
+                            "+warn_keywords"]),
+        ?OK([])).
+
+features(Config) when is_list(Config) ->
+    Run = run_fun(Config),
+
+    %% Simple compile - baseline for this file.  Really needed?
+    %% It uses atoms that are keywords in  experimental features.
+    Run("ignorant.erl",
+        [],
+        ?OK([])),
+
+    %% Enable and disable the same feature on the command line to
+    %% check that the ordering semantics is honoured.
+    Run("ignorant.erl",
+        options([longopt(enable, experimental_ftr_1),
+                 longopt(disable, experimental_ftr_1)]),
+        ?OK([])),
+
+    %% As above, with +options
+    Run("ignorant.erl",
+        options([plusopt(enable, experimental_ftr_1),
+                 plusopt(disable, experimental_ftr_1)]),
+        ?OK([])),
+
+    %% Mixed options
+    Run("ignorant.erl",
+        options([longopt(enable, experimental_ftr_1),
+                 plusopt(disable, experimental_ftr_1)]),
+        ?OK([])),
+
+    %% Mixed options
+    Run("ignorant.erl",
+        options([plusopt(enable, experimental_ftr_1),
+                 longopt(disable, experimental_ftr_1)]),
+        ?OK([])),
+
+    %% Enable a feature and see errors
+    Run("ignorant.erl",
+        options([longopt(enable, experimental_ftr_1)]),
+        ?NOTOK([syntax(ifn),
+                skip_lines,
+                syntax(ifn),
+                skip_lines])),
+
+    %% Enable another feature and see other errors
+    Run("ignorant.erl",
+        options([longopt(enable, experimental_ftr_2)]),
+        ?NOTOK([syntax(until),
+                skip_lines,
+                syntax(until),
+                skip_lines,
+                syntax(while),
+                skip_lines,
+                syntax(while),
+                skip_lines,
+                syntax(until),
+                skip_lines])),
+
+    %% Enable both features and see even more errors
+    Run("ignorant.erl",
+        options([longopt(enable, experimental_ftr_1),
+                 longopt(enable, experimental_ftr_2)]),
+        ?NOTOK([syntax(until),
+                skip_lines,
+                syntax(until),
+                skip_lines,
+                syntax(ifn),
+                skip_lines,
+                syntax(while),
+                skip_lines,
+                syntax(until),
+                skip_lines,
+                syntax(ifn),
+                skip_lines])),
+
+    ok.
+
+features_macros(Config) when is_list(Config) ->
+    {Run, _, OutDir} = run_fun_dirs(Config),
+
+    Run("f_macros.erl", "", ?OK([])),
+
+    {ok, Peer0, Node0} =
+        ?CT_PEER(#{args => ["-pa", OutDir],
+                   connection => 0}),
+    Call = fun(Node, Fun) ->
+                   rpc:call(Node, f_macros, Fun, [])
```

Comment:
In new code, we should always use `erpc` instead of `rpc`.

Reply:
Ok, will fix.